test/e2e/generated/BUILD

```diff
@@ -13,28 +13,8 @@ load(
 go_library(
     name = "go_default_library",
     srcs = [
+        "bindata.go",
         "main.go",
-        ":bindata",
```

Comment:
This change reverts non autogenerated build changes. It looks like you messed up a rebase. For the most part BUILD file changes should make sense in the context of the change you made. This doesn't make sense. If you see nonsensible changes please ping me.